croncert-config.yml

```diff
@@ -1786,6 +1835,59 @@ scrapers:
   ##########
   # Vevey
   ##########
+  - name: BoutDuMonde
+    url: https://www.leboutdumonde.ch/
+    item: section.list_program > ul.events > li.event.has_thumb
+    fields:
+      # static
+      - name: location
+        value: "Le Bout Du Monde"
+      - name: city
+        value: Vevey
+      - name: "sourceUrl"
+        value: "https://www.leboutdumonde.ch"
+      - name: type
+        value: concert
+      # dynamic
+      - name: date
```

Comment:
I had to add some extra config to properly extract date and time.

Reply:
Thanks. Sometimes this stuff is pretty subtile, and I'm still learning it.